### PR TITLE
Fire Weather Map Updates

### PR DESCRIPTION
**Pull request by hinzed1127**
## Proposed changes

- Integrates Contentful with the Fire Weather Map
   - Added a [single-use content type](https://app.contentful.com/spaces/pc5e1rlgfrov/content_types/fireDangerMap/fields) for that page.
   - [Content content using that content type](https://app.contentful.com/spaces/pc5e1rlgfrov/entries/7fiQCwkywcWi7lBy0dayqd?focusedField=title)

## Screenshots

Notice that it gets included in the navigation side bar:
Screenshot 2024-10-31 at 9 49 10 PM

Description rich text served from Contentful:
Screenshot 2024-10-31 at 9 49 20 PM

## Acceptance criteria validation

- [x] Map route exists as page details in Contentful
- [x] Description for the map lives in and is served from Contentful

### Alternate solutions

- In theory, we could use a core content page, since this is a subset of those properties, but I think it's better to explicitly restrict it to just the description alongside the map.

### CODE REVIEW

File: src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.server.ts
Comment (LouisFettet) [inline src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.server.ts:17]: This could have an adverse side-effect if there's more than one entry erroneously created; I'd recommend doing something like we did on `/licensing-permits` where we look up the  pageMetadata ID based on the route (and then query for the one entry instead of a collection): https://github.com/la-ldaf/ldaf-site/blob/9aa03aa9241b5f32bad23bbb8245454944269393/src/routes/licensing-permits/%2Bpage.server.ts#L95

File: src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.svelte
Comment (LouisFettet) [inline src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.svelte:114]: I think we have an off-by-one error here; I'm seeing the following on the preview:
* green: medium
* yellow: very high
* orange: extreme
* red: undefined

**Review by LouisFettet — COMMENTED** [review]
Noticed two issues that I've noted in code review that we should look into before merging.

Additionally, the page heading is an `h2`, but it should be an `h1`. I'm unsure where that's getting set (I assume it's by the Plot?).

**Review by LouisFettet — APPROVED** [review]
Pre-approving so I don't block you, but yeah I think at the least we need to handle the issue with the heading and the labels that appear on the parishes before merging.

File: src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.svelte
Comment (hinzed1127) [reply src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.svelte:114]: ah, yup. forgot about the array index offset. easy fix.

**Pull request merged**
